src/components/FileUploadForm.vue

```diff
@@ -180,7 +180,7 @@ export default Vue.extend({
       this.uploading = true;
 
       try {
-        await api.uploadTable(workspace, fileName, {
+        await api.uploadNetwork(workspace, fileName, {
```

Comment:
Why was this changed to `uploadNetwork`?

Reply:
This is the component that handles uploading files to create networks, so I switched it to use the new separate function.  TableDialog.vue takes care of uploading files to make tables.